src/main/java/com/example/bookshop/dto/UserLoginResponseDto.java

```diff
@@ -0,0 +1,10 @@
+package com.example.bookshop.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
```

Comment:
```suggestion
```

Read doc about `@Data`
`Equivalent to @Getter @Setter @RequiredArgsConstructor @ToString @EqualsAndHashCode`

Reply:
YES! But I need @AllArgsConstructor, because if I use only @Data I have the problem like this:

java: constructor UserLoginResponseDto in class com.example.bookshop.dto.UserLoginResponseDto cannot be applied to given types;
  required: no arguments
  found:    java.lang.String